src/components/ItemTable/ItemAvailability/ContactALibrarian.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button } from "@nypl/design-system-react-components"
+import { useContext } from "react"
+import { FeedbackContext } from "../../../context/FeedbackContext"
+import type { ItemMetadata } from "../../../types/itemTypes"
+import type Item from "../../../models/Item"
+
+const ContactALibrarian = ({
+  item,
+}: {
+  item: {
```

Comment:
Not necessary but just showing another way to type this. This is derived from the `Item` class when it's used as a class. You should be able to do `Pick<Item, "id" | "barcode" | "callNumber" | "bibId">`.

Reply:
And also reuse it in `NotAvailable` if you give this its own name.

Reply:
thats a good pattern to keep in mind! i kept going back and forth between passing the whole item and just the needed properties. Pick would have been useful.